### DISCUSSION TITLE
List of Ethereum chains is unordered in `graph init`

### DISCUSSION THREAD

**Discussion by schmidsi** [graphprotocol/graph-tooling#1616] [Ideas]
When I run `graph init` and want to initialise an EVM subgraph, the list of possible networks seems arbitrary:

<img width="850" alt="Screenshot 2024-03-23 at 12 29 37" src="https://github.com/graphprotocol/graph-tooling/assets/103280/4a16bd46-9267-4071-968f-a69dc0ac38ea">

It is especially confusing as some testnets are next to their mainnets, and some aren't. See `base` and `base-sepolia` for example.

I suggest that we order it like this:
1. Ethereum mainnet
2. Rest, alphabetically ordered

```
? Ethereum network … 
  mainnet
  arbitrum-goerli
  arbitrum-one
  arbitrum-sepolia
  arweave-mainnet
  astar-zkevm-mainnet
  astar-zkevm-sepolia
  aurora
  aurora-testnet
  avalanche
  base
  base-sepolia
  base-testnet
  blast-mainnet
  blast-testnet
  boba
  bsc
  celo
  celo-alfajores
  chapel
  clover
  cosmoshub-4
  fantom
  fantom-testnet
  fuji
  fuse
  gnosis
  goerli
  harmony
  holesky
  linea
  linea-goerli
  matic
  mbase
  moonbeam
  moonriver
  mumbai
  near-mainnet
  near-testnet
  optimism
  optimism-goerli
  optimism-sepolia
  osmo-test-4
  osmosis-1
  poa-core
  polygon-zkevm
  polygon-zkevm-testnet
  scroll
  scroll-sepolia
  sei-testnet
  sepolia
  theta-testnet-001
  zksync-era
  zksync-era-sepolia
  zksync-era-testnet
```

**Comment by saihaj** [comment]
Great idea! created https://github.com/graphprotocol/graph-tooling/issues/1618

**Comment by yash251** [comment]
I think we should also update the name `matic` to `polygon` or `polygon-mainnet` as new devs wouldn't be familiar with `matic` name
